base/string_utils.cpp

```diff
@@ -280,6 +280,20 @@ bool StartsWith(std::string const & s1, char const * s2) { return (s1.compare(0,
 
 bool StartsWith(std::string const & s1, std::string const & s2) { return (s1.compare(0, s2.length(), s2) == 0); }
 
+bool EndsWith(UniString const & s1, UniString const & s2)
+{
+  if (s1.size() < s2.size())
+    return false;
+
+  size_t const start = s1.size() - s2.size();
```

Comment:
nit: std::equal would be more in line with the implementations of other overrides

Reply:
done